sweep.py

```diff
@@ -128,13 +129,7 @@ def Bs(ix):
 
 #Apply boundary conditions, allow sliding along a vertical axis in the middle
 ix_i = int(nDOF/3/2)*3 #The middle vertical DOF
-ix = range(ix_i+1,ix_i+3)
-K = np.delete(K, ix, axis=0)
```

Comment:
Is this work like as deallocate in fortran?
I comment outed on my workspace but it must keep work on if you need these treatment...